agents1/LiarAgent.py

```diff
@@ -14,3 +15,66 @@ class LiarAgent(GenericAgent):
     def __init__(self, settings:Dict[str,object]):
         super().__init__(settings, Phase.PLAN_PATH_TO_CLOSED_DOOR)
         self._lying_prob = 0.80
+
+    def lie_message(self, message):
+        """
+            generates a random lie based on the message received
+
+        Args:
+            message: str that MessageBuilder can process
+
+        Note:
+            This method can only change:
+            - room name
+            - block shape
+            - block colour
+            - location
+
+        Returns:
+            Message
+        """
+        
+        content = MessageBuilder.process_message(message)
+        keys = list(content.keys())
+
+        if 'room_name' in keys:
+            content['room_name'] = f'room_{(int(content["room_name"][-1]) + 1) % 9}'  # increment the room
+
+        if 'visualization' in keys:
```

Comment:
looks like you dont lie about the size

Reply:
Yes, i felt that we do not need to lie about the size since it doesn't change and we already modify the colour and shape